docs/snippets/web-components/my-component-story-use-globaltype-backwards-compat.ts.mdx

```diff
@@ -0,0 +1,19 @@
+```ts
+// MyComponent.stories.ts
+
+import { html } from 'lit-html';
+
+import { Story } from '@storybook/web-components';
```

Comment:
I think `Story` is for CSF 2, CSF 3 must use `StoryObj`:

```suggestion
import { StoryObj } from '@storybook/web-components';
```

Reply:
Thanks for that, I caught up on it as well and I'm updating the example. I'll push a commit in a bit